### ISSUE TITLE
New `0-new-issue.yml` issue template creates needless headings in issue bodies

### ISSUE THREAD

**Issue by not-my-profile** [whatwg/spec-factory#61]
With the new issue template every issue created starts with:

```
### What is the issue with the HTML Standard?
```

which is rather redundant. Can we somehow disable this?

**Comment by annevk** [comment]
Yeah, I noted this in Chat a while back. Upvote https://github.com/orgs/community/discussions/63402#discussioncomment-7113247 and hope GitHub gets to it is the best I could think of. (I've also let someone at GitHub know.)

(I think the trade-off is worth it, but I agree it would be nice if this were better.)

**Comment by Lishkk** [comment]
> With the new issue template every issue created starts with:
> 
> ```
> ### What is the issue with the HTML Standard?
> ```
> 
> which is rather redundant. Can we somehow disable this?

**Comment by Lishkk** [comment]
> Yeah, I noted this in Chat a while back. Upvote https://github.com/orgs/community/discussions/63402#discussioncomment-7113247 and hope GitHub gets to it is the best I could think of. (I've also let someone at GitHub know.)
> 
> (I think the trade-off is worth it, but I agree it would be nice if this were better.)

**Comment by Lishkk** [comment]
> With the new issue template every issue created starts with:
> 
> ```
> ### What is the issue with the HTML Standard?
> ```
> 
> which is rather redundant. Can we somehow disable this?

**Comment by Lishkk** [comment]
> Yeah, I noted this in Chat a while back. Upvote https://github.com/orgs/community/discussions/63402#discussioncomment-7113247 and hope GitHub gets to it is the best I could think of. (I've also let someone at GitHub know.)
> 
> (I think the trade-off is worth it, but I agree it would be nice if this were better.)
